CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 2.3.0 - 2025-03-04
+- [GRD-872](https://jira.oicr.on.ca/browse/GRD-872) - Update to add RNA mode option, setting up DNA and -ERC GCVF by default
 ## 2.2.0 - 2024-06-25
 [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
```

Comment:
Monica,you may want to re-format this file according to the guidelines specified in https://keepachangelog.com/en/1.0.0/ or use some properly formatted CHANGELOG.md (for example, https://github.com/oicr-gsi/mrdetect/blob/GDI-3780/CHANGELOG.md)